data/content.go

```diff
@@ -1579,13 +1690,30 @@ var ContentHeuristics = map[string]*Heuristics{
 	".stl": &Heuristics{
 		rule.Or(
 			rule.MatchingLanguages("STL"),
-			regex.MustCompileRuby(`\A\s*solid(?=$|\s)(?m:.*?)\Rendsolid(?:$|\s)`),
+			regex.MustCompileRuby(`\A\s*solid(?=$|\s)(?:.|[\r\n])*?^endsolid(?:$|\s)`),
```

Comment:
There is currently a patch to v7.26.0 that is on the way: https://github.com/github-linguist/linguist/pull/6518

I'm wondering if it's worth waiting for this patch to be incorporated into a future Linguist version or if the change suggested there could be simply applied manually by changing `\A\s*solid(?=$|\s)(?:.|[\r\n])*?^endsolid(?:$|\s)` to `\A\s*solid(?:$|\s)[\s\S]*^endsolid(?:$|\s)`

Reply:
Thanks for digging into it! 

I would wait for a patch to propagate instead of apply re-writes at runtime (as Enry policy is not to change the original regexps during code generation).

That being said, I'm happy to accept the PRs doing a (temporary) re-write at runtime, if we want a release before 7.26.1 is out.

Reply:
Seems like the next release is coming soon (most likely early next week), so I'm assuming it's better to wait for it and have everything in order + the new changes coming in v7.27.0.